dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -861,6 +870,28 @@ struct VolumeLockHolderArgs {
     uuid: Uuid,
 }
 
+#[derive(Debug, Args, Clone)]
+#[clap(group(
+ArgGroup::new("volume-reference-group")
+    .required(true)
+    .args(&["ip", "net", "read_only_region", "region_snapshot"])
+))]
+struct VolumeReferenceArgs {
+    #[clap(long, conflicts_with_all = ["net", "read_only_region", "region_snapshot"])]
+    ip: Option<std::net::Ipv6Addr>,
+
+    #[clap(long, conflicts_with_all = ["ip", "read_only_region", "region_snapshot"])]
+    net: Option<oxnet::Ipv6Net>,
+
+    #[clap(long, conflicts_with_all = ["ip", "net", "region_snapshot"])]
+    read_only_region: Option<Uuid>,
+
+    /// Provide dataset, region, and snapshot ID in a string, delimited by
```

Comment:
This help comment here is what I need to see.

From the help output:
`--region-snapshot <REGION_SNAPSHOT> `
That makes it seem like I only need a single item.  Is this part done by clap?

Reply:
yeah. with 996da6e this now looks like `--region_snapshot <REGION_SNAPSHOT> <REGION_SNAPSHOT> <REGION_SNAPSHOT>` and I don't know how to fix it :-1:

Reply:
fixed in dea7c6968